src/test/package/nuget/NuGetTest/TestApp/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Windows.CppWinRT" version="2.0.190712.2" targetFramework="native" />
```

Comment:
This will test the published NuGet package version N-1, so regressions could still slip thru.  Instead, we should remove all the packages.config files and restructure all the vcxproj files to import Microsoft.Windows.CppWinRT.props and Microsoft.Windows.CppWinRT.targets via relative paths "as if" the Nuget package had been added.  I did this originally for the fast forwarder test project - see 
src/tool/cppwinrt/test_fast_fwd/test_fast_fwd.vcxproj in PR #344